CHANGELOG.md

```diff
@@ -5,9 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- [#54](https://github.com/seznam/slo-exporter/pull/54) Use strict unmarshal in relabel module
```

Comment:
I'd move this to the `Changed` section since it changes behavior?

Reply:
fixed